crates/solver/src/auction_preprocessing.rs

```diff
@@ -2,8 +2,10 @@
 
 use crate::liquidity::LimitOrder;
 
-// vk: I would like to extend this to also check that the order has minimum age but for this we need
-// access to the creation date which is a more involved change.
 pub fn has_at_least_one_user_order(orders: &[LimitOrder]) -> bool {
     orders.iter().any(|order| !order.is_liquidity_order)
 }
+
+pub fn has_at_least_one_mature_order(orders: &[LimitOrder]) -> bool {
```

Comment:
Logic here is to skip auctions that don't contain at least 1 mature order. 
Note that I didn't want to filter out all non-mature orders since they can be part of a valid settlement by `association` maturity rule.